service/api_logic/games_logic.py

```diff
@@ -45,9 +55,13 @@ def get_games_today(
 
     offset, limit = pagination.get_pagination()
     if offset is not None and limit is not None:
+        api_logic_logger.info(f"Applying pagination: offset={offset}, limit={limit}")
```

Comment:
I don't think we need to log this

Keep in mind that YOU will be hostng this app. And you'll be getting thousands of different requests every hour (worst case estimation). I don't think you will need all this stuff in your log files wasting memory